src/test/java/org/jabref/logic/importer/fetcher/DOABFetcherTest.java

```diff
@@ -0,0 +1,130 @@
+                " open licenses, user communities, measuring usage, developing countries and the effects on" +
+                " citations and social media.");
+        Ronald_Snijder.setField(StandardField.LANGUAGE, "English");
+        Ronald_Snijder.setField(StandardField.KEYWORDS, "Open Access,Monographs,OAPEN Library," +
+                "Directory of Open Access Books");
+        Ronald_Snijder.setField(StandardField.PUBLISHER, "Amsterdam University Press");
+
+        Andrew_Perrin = new BibEntry();
+        Andrew_Perrin.setField(StandardField.EDITOR, "Andrew Perrin and Loren Stuckenbruck");
+        Andrew_Perrin.setField(StandardField.TITLE, "Four Kingdom Motifs before and beyond the Book of Daniel");
+        Andrew_Perrin.setField(StandardField.TYPE, "book");
+        Andrew_Perrin.setField(StandardField.DOI, "10.1163/9789004443280");
+        Andrew_Perrin.setField(StandardField.PAGES, "354");
+        Andrew_Perrin.setField(StandardField.YEAR, "2020");
+        Andrew_Perrin.setField(StandardField.URI, "https://directory.doabooks.org/handle/20.500.12854/68086");
+        Andrew_Perrin.setField(StandardField.ABSTRACT, "The four kingdoms motif enabled writers of various " +
+                "cultures, times, and places, to periodize history as the staged succession of empires " +
+                "barrelling towards an utopian age. The motif provided order to lived experiences under empire" +
+                " (the present), in view of ancestral traditions and cultural heritage (the past), and inspired" +
+                " outlooks assuring hope, deliverance, and restoration (the future). Four Kingdom Motifs before" +
+                " and beyond the Book of Daniel includes thirteen essays that explore the reach and redeployment" +
+                " of the motif in classical and ancient Near Eastern writings, Jewish and Christian scriptures," +
+                " texts among the Dead Sea Scrolls, Apocrypha and pseudepigrapha, depictions in European" +
+                " architecture and cartography, as well as patristic, rabbinic, Islamic, and African writings " +
+                "from antiquity through the Mediaeval eras. Readership: Advanced students and scholars of the " +
+                "textual formation, apocalyptic theology, and historiographies of the book of Daniel and its " +
+                "diverse reception by writers and communities.");
+        Andrew_Perrin.setField(StandardField.LANGUAGE, "English");
+        Andrew_Perrin.setField(StandardField.KEYWORDS, "Religion");
+        Andrew_Perrin.setField(StandardField.PUBLISHER, "Brill");
+
+    }
+
+    @Test
+    public void TestGetName() {
+        assertEquals("DOAB", fetcher.getName());
+    }
+
+    @Test
+    public void TestPerformSearch() throws FetcherException {
+        List<BibEntry> entries;
+        entries = fetcher.performSearch("i open fire");
+        assertFalse(entries.isEmpty());
```

Comment:
assertFalse and assertTrue make it hard to see why a test fails. In thise casse it's better you compare the entry collections.
e.g.  assertEquals(Collections.singletonlist(expectedEntry),entries)

Reply:
yes, sure